src/format.rs

```diff
@@ -281,6 +281,11 @@ impl FormatBuilder {
         FormatBuilder { format: Box::new(format) }
     }
 
+    /// Consumes the builder and returns the generated `TableFormat`
+    pub fn into_table_format(self) -> TableFormat {
+        *self.format
+    }
```

Comment:
Do you think so ? I don't really know what to think. I mean there are many crates which to that, and implements `From` or `Into` only for convenience. Some kind of aliases to the real functions.

But if you think only having `From` and `Into` impl and no dedicated functions, I'm also agree with that.